packages/web-components/src/components/va-search-input/va-search-input.tsx

```diff
@@ -120,7 +136,14 @@ export class VaSearchInput {
    * Fires a submit and analytics events.
    */
   private handleSubmit = () => {
-    this.inputRef.dispatchEvent(
+    const form = this.el.shadowRoot.querySelector('form');
+    if (form) {
+      form.onsubmit = (event) => {
+        event.preventDefault();
+      }
+    }
+
+    this.el.dispatchEvent(
       new SubmitEvent('submit', {
```

Comment:
Originally the component was dispatching the submit event using`this.inputRef` (the `input` element) but because the input element is now a child of `form`, the event would bubble up and be prevented by the parent element `form`  (at line 142). So the submit event would never actually fire. 

To fix that, I changed the element we are dispatching off of to the component itself `this.el`.